src/sass/_bootstrap-variables.scss

```diff
@@ -366,6 +366,8 @@ $caret-spacing:               $caret-width * 0.85 !default;
 
 $transition-base:             all 0.2s ease-in-out !default;
 $transition-fade:             opacity 0.15s linear !default;
+// CUSTOM SUMOCODERS
```

Comment:
Would/t it be better to follow the code styling that is already used, and write the comment in lowercase. Maybe also explain why this variable needs to exists and where and why it is used.